### DISCUSSION TITLE
How do I add my token?

### DISCUSSION THREAD

**Discussion by Inedhelplol** [marioparaschiv/nitro-sniper#107] [Q&A] [answered]
Once I run it, it says "No main account token provided, exiting process......" I've got my token but I dont know where to put it.
<img width="578" alt="Screenshot 2021-05-13 at 12 57 16 PM" src="https://user-images.githubusercontent.com/84116889/118079440-cc863a00-b3ea-11eb-9bfb-f7de538afccc.png">

**Comment by MrD4rio** [comment, marked as answer]
You have to put your token(s) in the .env file that you find in the nitro-sniper folder. That's the config file, more infos about it can be found there: https://github.com/slow/nitro-sniper/wiki
